documentation-site/pages/getting-started/comparison.mdx

```diff
@@ -0,0 +1,44 @@
+<!--
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+import Layout from '../../components/layout';
+import {Table} from 'baseui/table';
+
+export default Layout;
+
+# How Base UI compares with other component libraries
+
+We understand that before start using a component library and invest
+time to learn it, you want to better understand how it is designed
+and what constraints you'll have to work with. On this page,
+we'd like to provide you with this information.
+
+## Technical overview
+
+The table below shows you a high-level comparison of the most popular
+React component libraries and Base UI. While we try to do our best
+to keep it up-to-date, it may have incorrect information. If you
+notice that, please let us know in a GitHub issue, or send a Pull Request fixing it!
+
+<Table columns={['Name', 'CSS Engine', 'Customization', 'Size (gzipped)']} data={[
+  ['Base UI', 'CSS-in-JS, using Styletron', 'Through an extensive overrides interface', <a href="https://bundlephobia.com/result?p=baseui@5.13.2">~5kB</a>],
```

Comment:
It's misleading to say that Base UI has `5kB`. We don't export any components from the `index.js` so bundlephobia considers only the size of some helper functions (withTheme, styled, etc...)

Reply:
you would have to import all components into a single file and bundle it